src/lang/terms/is_a.pl

```diff
@@ -183,7 +183,7 @@
   assert_true(subproperty_of(test:'hasBrother', test:'hasSibling')).
 
 % expect instantiation error as is_a expects a ground variable
-test("ask A is a Person", [throws(error(instantiation_error, _))]) :-
-  is_a(A, dul:'Person').
+test("ask _ is a Person", [throws(error(instantiation_error, _))]) :-
+  is_a(_, dul:'Person').
 
 :- end_tripledb_tests(lang_is_a).
```

Comment:
not really important for the pl source files, but well github complains about missing newline at the end of the code (also below). Could you add it?

Reply:
Done